servicetalk-grpc-api/src/main/java/io/servicetalk/grpc/api/GrpcRoutes.java

```diff
@@ -208,7 +208,7 @@ protected GrpcService newServiceFromRoutes(final AllGrpcRoutes routes) {
             final String path, final GrpcExecutionStrategy executionStrategy,
             final StreamingRoute<Req, Resp> route, final Class<Req> requestClass,
             final Class<Resp> responseClass, final GrpcSerializationProvider serializationProvider) {
-        routeBuilder.addStreamingRoute(path, null, route, requestClass, responseClass,
+        routeBuilder.addStreamingRoute(path, executionStrategy, route, requestClass, responseClass,
```

Comment:
good catch ... unit test here?

Reply:
I have created an issue #907 to add tests to verify the strategy used by routes as @idelpivnitskiy is working on it ATM.